src/server/system_services/bucket_server.js

```diff
@@ -251,6 +251,49 @@ function validate_pool_constraints({ mongo_pool, default_pool }) {
     }
 }
 
+function validate_replication(req, configured_buckets, min_required) {
+    const replication_policy = req.rpc_params.replication_policy;
+    // num of rules in configuration must be in defined limits
+    if (replication_policy.length > config.BUCKET_REPLICATION_MAX_RULES ||
+        (min_required && replication_policy.length < config.BUCKET_REPLICATION_MIN_RULES)) throw new RpcError('INVALID_REPLICATION_POLICY');
```

Comment:
we can just check that `replication_policy.length > 0`

Reply:
if we don't want to have the minimum in config, yes